about.html

```diff
@@ -17,10 +17,11 @@
     <div class="toptop">
```

Comment:
Your header/nav looks fine.  You can simplify this some and remove a couple of divs and class names.  But, it is not needed.